packages/react-core/src/evm/contexts/thirdweb-auth.tsx

```diff
@@ -36,28 +35,28 @@ const ThirdwebAuthContext = createContext<ThirdwebAuthContext | undefined>(
 );
 
 export const ThirdwebAuthProvider: React.FC<
-  PropsWithChildren<{ value?: ThirdwebAuthConfig }>
-> = ({ value, children }) => {
-  const signer = useSigner();
+  PropsWithChildren<ThirdwebAuthConfig>
+> = ({ domain, authUrl, children }) => {
+  const wallet = useWallet();
 
   // Remove trailing slash from URL if present
   const authContext = useMemo(() => {
-    if (!value) {
+    if (!domain) {
       return undefined;
     }
 
     const context: ThirdwebAuthContext = {
-      ...value,
-      authUrl: value.authUrl?.replace(/\/$/, ""),
+      domain,
+      authUrl: authUrl?.replace(/\/$/, ""),
       auth: undefined,
     };
 
-    if (signer) {
-      context.auth = new ThirdwebAuth(new SignerWallet(signer), value.domain);
+    if (wallet) {
+      context.auth = new ThirdwebAuth(wallet, domain);
```

Comment:
I think we need to do new EthersWallet here instead of using the `useWallet` hook